Session4_v2.Rmd

```diff
@@ -0,0 +1,354 @@
+---
+source: Rmd
+title: "GIS operation on vector and raster data"
+author: "Clémentine Cottineau"
+teaching: 75
+exercises: 45
+---
+
+# Objectives:
+- Import, select, and visualise OSM vector data
+- Learn about basic GIS operation: spatial buffers, centroids, area calculation and vector/raster intersection
+
+
+
+```{r, include=FALSE}
+source("bin/chunk-options.R")
+knitr_fig_path("04")
+```
+
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+	echo = TRUE,
+	message = FALSE,
+	warning = FALSE
+)
+```
+
+```{r lib}
+library(dplyr)
+library(sf)
+library(ggplot2)
+library(osmdata)
+library(leaflet)
+```
+
+# Part 1. Import vector data from Open Street Map
+
+## What is Open Street Map?
+
+Open Street Map (OSM) is a collaborative project which aims at mapping the world and sharing geospatial data in an open way. Anyone can contribute, by mapping geographical objects their encounter, by adding topical information on existing map objects (their name, function, capacity, etc.), or by mapping buildings and roads from satellite imagery (cf. [HOT: Humanitarian OpenStreetMap Team](https://www.hotosm.org/)).
+
+This information is then validated by other users and eventually added to the common "map" or information system. This ensures that the information is accessible, open, verified, accurate and up-to-date.
+
+The result looks like this:
+![View of OSM web interface](fig/OSM1.png)
+The geospatial data underlying this interface is made of geometrical objects (i.e. points, lines, polygons) and their associated tags (#building #height, #road #secondary #90kph, etc.).
+
+## How to extract geospatial data from Open Street Map?
+
+### Bonding-box
+
+The first thing to do is to define the area within which you want to retrieve data, aka the *bounding box*. This can be defined easily using a place name and the function `getbb()` from the package `osmdata`. 
+
+> "This function uses the free Nominatim API provided by OpenStreetMap to find the bounding box (bb) associated with place names."
+
+We are going to look at *Brielle* together, but you can also work with the small cities of *Naarden*, *Geertruidenberg*, *Gorinchem*, *Enkhuizen* or *Dokkum*.
+
+```{r osm-bounding-box}
+assign("has_internet_via_proxy", TRUE, environment(curl::has_internet))
```

Comment:
Maybe this can be put into a separate code chunk with a side note to those who might get an internet connection error. An explanation would be useful for those who are following this tutorial outside the workshop.